spec/site_prism/page_objects/page/organisations/courses/back_new_modern_languages_page.rb

```diff
@@ -0,0 +1,11 @@
+module PageObjects
+  module Page
+    module Organisations
+      module Courses
+        class NewModernLanguagesPage < NewCourseBase
```

Comment:
why is that is called `BackNewLocationsPage` and this is called `NewModernLanguagesPage` :thinking: